### PR TITLE
Add several USDT, ETH and BTC representations

### PR DESCRIPTION
**Pull request by JohnnyWyles**
Adding several USDT, ETH and BTC representations on Osmosis from our asset list: https://github.com/osmosis-labs/assetlists/blob/main/osmosis-1/generated/frontend/assetlist.json

### CODE REVIEW

**Review by songwongtp — DISMISSED** [review]

File: data/assets.json [hunk 1]
```diff
@@ -6874,6 +6874,108 @@
...
+    "id": {
+      "osmosis": {
+        "osmosis-1": "ibc/D105950618E47CA2AEC314282BC401625025F80A4F812808DEEBB1941C685575"
+      }
+    },
+    "logo": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/usdt.solana.pica.svg",
+    "name": "Solana USDT on Picasso",
+    "precision": 6,
+    "slugs": [],
+    "symbol": "solana.USDT.pica",
```
Comment (songwongtp) [inline data/assets.json:6888]: what is the difference between this and `USDT.sol.pica`?

https://app.osmosis.zone/assets/USDT.sol.pica?sellOpen=false&buyOpen=false

Comment (JohnnyWyles) [reply data/assets.json]: we changed the naming scheme around this time to go from
chain.Token.bridge to Token.chain.bridge
was just an older format, have fixed

Comment (songwongtp) [reply data/assets.json]: does the ibc denom need an update too?

File: data/assets.json [hunk 2]
```diff
@@ -6874,6 +6874,108 @@
     "type": "native"
   },
   {
+    "coingecko": "tether",
+    "description": "Tether from Solana via Picasso",
+    "id": {
+      "osmosis": {
+        "osmosis-1": "ibc/D105950618E47CA2AEC314282BC401625025F80A4F812808DEEBB1941C685575"
```
Comment (songwongtp) [inline data/assets.json:6881]: should this be changed to this?
`ibc/0233A3F2541FD43DBCA569B27AF886E97F5C03FC0305E4A8A3FAC6AC26249C7A`

Comment (JohnnyWyles) [reply data/assets.json:6881]: Yes, sorry, I copied the Source denom from our asset lists

**Review by songwongtp — APPROVED** [review]

**Pull request merged**